x/ibc/04-channel/keeper/timeout.go

```diff
@@ -266,28 +266,6 @@ func (k Keeper) TimeoutOnClose(
 		return err
 	}
 
-	k.deletePacketCommitment(ctx, packet.GetSourcePort(), packet.GetSourceChannel(), packet.GetSequence())
```

Comment:
I think this was an unreachable bug that we didn't close ordered channels upon timeout